react/FilePicker/FilePickerBreadcrumb.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+
+import Typography from '../Typography'
+import Icon from '../Icon'
+import RightIcon from '../Icons/Right'
+import useBreakpoints from '../hooks/useBreakpoints'
+
+const useStyles = makeStyles(() => ({
+  icon: {
+    color: 'var(--actionColorActive)',
+    margin: '0 0.25rem'
+  },
+  previousPath: {
+    color: 'var(--actionColorActive)',
+    cursor: 'pointer'
+  }
+}))
+
+const FilePickerBreadcrumb = ({ path, onBreadcrumbClick }) => {
+  const { isMobile } = useBreakpoints()
+  const classes = useStyles()
+  const hasPath = path && path.length > 0
+
+  const navigateTo = useCallback(folder => () => onBreadcrumbClick(folder), [
+    onBreadcrumbClick
+  ])
+
+  return (
+    <Typography variant="h4" className={'u-flex u-flex-items-center'}>
+      {hasPath
+        ? isMobile
+          ? path[path.length - 1].name
+          : path.map((folder, idx) => {
+              if (idx < path.length - 1) {
+                return (
+                  <Fragment key={idx}>
+                    <span
+                      className={classes.previousPath}
+                      onClick={navigateTo(folder)}
+                    >
+                      {folder.name}
+                    </span>
+                    <Icon icon={RightIcon} className={classes.icon} />
+                  </Fragment>
+                )
+              } else {
+                return <span key={idx}>{folder.name}</span>
+              }
+            })
+        : null}
+    </Typography>
+  )
+}
+
+FilePickerBreadcrumb.propTypes = {
+  path: PropTypes.array,
+  onBreadcrumbClick: PropTypes.func
```

Comment:
we could add one test to check the click on the breadcrumb 🙏

Reply:
I don't want to take more time on this part of the component, it will certainly change soon